Square.cpp

```diff
@@ -1,15 +1,37 @@
 #ifndef Square_cpp
 #define Square_cpp
+
 #include <iostream>
-#include "Draw.cpp"
+#include "Shape.cpp"
+#include "Area.cpp"
+#include "Input.cpp"
+
 using namespace std;
 
-class Square : public Draw
+class Square : public Shape , public Input
 {
 public:
     void drawShape() override
     {
         cout << "square" << endl;
     }
+
+    int inputLength() override 
+    {
+        int length;
+        cout << "Enter Length" << endl;
+        cin >> length;
+        return length;
+    }
+    int CalculateArea() override
```

Comment:
Same as before `CalculateArea` is doing more than it promises to by its name by doing two jobs.